shared-module/displayio/Shape.c

```diff
@@ -49,10 +49,16 @@ void common_hal_displayio_shape_construct(displayio_shape_t *self, uint32_t widt
     self->half_height = height;
 
     self->data = m_malloc(height * sizeof(uint32_t), false);
+    //for (uint16_t i = 0; i < height; i++) {
```

Comment:
Do we need to keep this commented out line?

Reply:
Deleted unnecessary comment.  Thanks for testing it and catching this leftover comment.